app/(gcforms)/[locale]/(form administration)/form-builder/[id]/preview/BackButton.tsx

```diff
@@ -4,7 +4,13 @@ import { Button } from "@clientComponents/forms";
 import { Language } from "@lib/types/form-builder-types";
 
 // Must be placed withing context of the GCFormsContext.Provider
-export const BackButton = ({ language }: { language: Language }) => {
+export const BackButton = ({
+  language,
+  callback,
+}: {
+  language: Language;
+  callback?: () => void;
```

Comment:
Wondering if there's a better name for this maybe just onClick?

callback makes me think of something that's going to happen after a function call happens which isn't exactly what's happing here.